conf/main/grakn.properties

```diff
@@ -100,6 +100,8 @@ log.dirs=./logs/
 log.level=INFO
 
 ############################# Persisted Knowledge Base Configuration #############################
+# Directory in which the data will be persisted
+storage.db-dir=db/
```

Comment:
I think this should just be`db-dir`, meaning that we have 1 directory setting for everything.

For example, if people set `db.dir=/opt/grakn-data`, Cassandra data would be in `/opt/grakn-data/cassandra` (and in the future Redis data would be in `/opt/grakn-data/redis`